Doc/c-api/veryhigh.rst

```diff
@@ -187,19 +187,14 @@ the same library that the Python runtime is using.
       :c:func:`PyMem_Malloc` or :c:func:`PyMem_Realloc`.
 
 
-.. c:function:: struct _node* PyParser_SimpleParseString(const char *str, int start)
```

Comment:
I am missing something, why are `PyParser_SimpleParseString` and `PyParser_SimpleParseFile` being removed from the docs?

Reply:
You're not missing anything. This was badly committed. I had removed them for some reason, but forgot to add them back, after I realised they shouldn't go. Fixed in https://github.com/python/cpython/pull/21012/commits/ad23355bf0d821e39c980276ee1d3eeed3edad4b.